tests/app/test_modifier.py

```diff
@@ -47,17 +52,37 @@ def populate_database(self, record_count=0, mark_as_read=True):
                             VALUES('{EventsApi.READ.value}', '{msg_id}', '2017-02-03 00:00:00')'''
                     con.execute(query)
 
+        return msg_id
```

Comment:
Relocate the `msg_id = str(uuid.uuid4())` to outside the `for` to resolve the reference error?

Reply:
Fixed, but in a slightly different way